docs/collections/documents/geospatial/tutorial.md

```diff
@@ -75,130 +75,137 @@ Let's assume your
-    cursor = client.execute_query(near_query)
-    docs = [document for document in cursor]
-    pp.pprint(docs)
-
-  #-------------------------------------------------------------------#
-  
-    # # Step7: Delete Data
-    print("\n7. DELETE_DATA: region: {}, collections: {}, {}".format(global_url, collection_shops, collection_transit))
-    shopCol.truncate()
-    transitCol.truncate()
-    #client.delete_collection(collection_name)
+""" This file is a demo of using geoLocations on collections """
+import pprint
+from c8 import C8Client
+
+# Variables - Connection
+URL = "gdn.paas.macrometa.io"
+API_KEY = "my API key" #Change to my API key
+GEO_FABRIC = "_system"
+
+
+# Variables - DB
+collection_transit = "transit"
+collection_shops = "shops"
+
+# Variable - Requests locations
+latitude2 = 37.3972452334285 # Sunnyvale Elementary School
+longitude2 = -121.99073796514953 # Sunnyvale Elementary School
+
+# Variables - Data
+shops_data = [
+    { "shop": "Walgreens", "address": "San Jose, CA", "coordinate": [ 37.2333253, -121.6846349 ] },
+    { "shop": "Costco", "address": "San Jose, CA", "coordinate": [ 37.4282724, -121.9066238 ] },
+    { "shop":"Walmart", "address": "San Jose, CA", "coordinate":
+        [ 37.3361905, -121.8905833 ] },
+    { "shop": "Bed Bath & Beyond", "address": "Mountain View, CA", "coordinate":
+        [ 37.4443293, -122.1598465] },
+    { "shop":"Best Buy", "address": "Mountain View, CA", "coordinate":
+        [ 37.35992431640625,-122.06830596923828 ] },
+    { "shop": "Target", "address": "Mountain View, CA", "coordinate":
+        [ 37.4443293, -122.1598465] },
+    { "shop": "Walgreens", "address": "Fremont, CA", "coordinate":
+        [ 37.48849288005677, -121.94342916254486 ] },
+    { "shop": "Costco", "address": "Almeda, CA", "coordinate":
+        [ 37.766039418486855, -122.2430211806723 ] },
+    { "shop": "Sweet Greens", "address": "San Francisco, CA", "coordinate":
+        [ 38.621725, -92.571271 ] }
+ ]
+
+transit_data = [
+    { "bus": "CA123", "stop": "Santa Clara, CA", "coordinate": [ 37.2333253, -121.6846349 ] },
+    { "bus": "CA233", "stop": "Milpitas, CA", "coordinate": [ 37.4282724, -121.9066238 ] },
+    { "bus": "CA433", "stop": "San Jose, CA", "coordinate": [ 37.3361905, -121.8905833 ] },
+    { "bus": "CA323", "stop": "Palo Alto, CA", "coordinate": [ 37.4443293, -122.1598465] },
+    { "bus": "CA523", "stop": "Mountain View, CA", "coordinate":
+        [ 37.35992431640625,-122.06830596923828 ] },
+    { "bus": "CA143", "stop": "Fremont, CA", "coordinate":
+        [ 37.48849288005677, -121.94342916254486 ] },
+    { "bus": "CA423", "stop": "Almeda, CA", "coordinate":
+        [ 37.766039418486855, -122.2430211806723 ] },
+    { "bus": "CA129", "stop": "San Francisco, CA", "coordinate": [ 38.621725, -92.571271 ] }
+]
+
+pp = pprint.PrettyPrinter(indent=4)
+
+# Step1: Open connection to GDN. You will be routed to closest region.
```

Comment:
Please put a space between the step and the number.

Step 1
Step 2

Reply:
done :)